File: Week 6/pset6/mario.py
# mario in python
from cs50 import get_int


def main():
    n = get_number()

    # loop
    for i in range(n):
        print(" "*(n-i-1), end="")
        print("#"*(i+1), end="")
        # print newline
        print()

# getting height


def get_number():
    n = 0
    while n < 1 or n > 8:
        n = get_int("Height: ")
    if n >= 1 or n <= 8:
        return n


main()

File: Week 1/pset1/mario/mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)
{

    int n;

    do
    {
        n = get_int("Height: ");            //get number
    }
    while (n < 1 || n > 8);                 // numbers between 1 and 8


    for (int g = 0; g < n; g++)             // first loop: line
    {
        for (int h = n - 1; h > g; h--)     //second loop: spaces
        {
            printf(" ");
        }
        for (int i = 0; i <= g; i++)        //third loop: hashes
        {
            printf("#");
        }
        printf("\n");
    }

}


